aerogear/mobile-security-service
# fix(swagger): Updated swagger doc to reflect real endpoint URL (AEROG…

…EAR-XXXX)

## Motivation

Endpoint URL was wrong in Swagger definition file.

## What

Updated the endpoint from `/apps/:id/versions/batch-disable` to `/apps/:id/versions/disable`.

## Why

Doc was outdated.

## Verification Steps
Add the steps required to check this change. Following an example.
 
Verify that [this line](https://github.com/aerogear/mobile-security-service/blob/42b4e26663de578ebbb7f0fa0bde7fdc3c51b231/api/swagger.yaml#L133) says `/apps/:id/versions/disable:`

## Checklist:


## Progress
